### PR TITLE
Use api.decred.org

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: www-root/content/js/brand.js [hunk 1]
```diff
@@ -11,8 +11,7 @@ $(document).ready(function() {
 	    viewport = $(window),
 
 	    // get json
-		APIdc = '/api/?c=dc',
-	    APIstats = '/api/?c=gcs',
```
Comment (alexlyp) [inline www-root/content/js/brand.js]: do you need to replace this one too?

Comment (dajohi) [inline www-root/content/js/brand.js]: Couldn't find APIstats used...

Comment (jolan) [inline www-root/content/js/brand.js]: Yeah gcs = get coin supply. It was more efficient to just calculate this in javascript so that is unused.

**Comment by alexlyp** [comment]
closing since #102 included these changes

**Pull request closed without merging**